omero/sysadmins/system-requirements.txt

```diff
@@ -64,20 +64,48 @@ The recommended client specification is:
 
 Large imports may require 4GB RAM.
 
+.. _client_directories:
+
 Client configuration
 ^^^^^^^^^^^^^^^^^^^^
 
 When performing some operations the clients make use of temporary file
-storage and log directories. By default these files are stored below
-the user's HOME directory in :file:`$HOME/omero/tmp`,
-:file:`$HOME/omero/log`, and :file:`$HOME/omero/sessions`
-(resp. :file:`$HOME\\omero\\tmp`, :file:`$HOME\\omero\\log`, and
-:file:`$HOME\\omero\\sessions`). If your home
-directory is stored on a network, possibly NFS mounted (or similar),
-then these temporary files are being written and read over the
-network. This can slow access down.  The :envvar:`OMERO_TEMPDIR`
-environment variable may used to set the directory used for temporary
-files, for example on fast local storage.
+storage and log directories. The table below indicates the default values for
+each directory and the environment variables allowing to set their locations:
+
+.. list-table::
+  :header-rows: 1
+
+  - * Client directory
+    * Environment variable
+    * Default location (UNIX)
+    * Default location (Windows)
+  - * OMERO user directory
+    * :envvar:`OMERO_USERDIR`
+    * :file:`$HOME/omero`
+    * :file:`%HOMEPATH%\\omero`
```

Comment:
Is it correct though?  I see a single use of HOMEPATH in etc/env.bat, but I can't see any obvious use of this batch file, making me wonder if it's used at all.
Also, see
```
components/tools/OmeroPy/src/omero/util/__init__.py:        homeprop = shell.SHGetFolderPath(0, shellcon.CSIDL_APPDATA, 0, 0)
```

Is it using APPDATA instead by default?

Note APPDATA is the "roaming profile" which is possibly not local, and is synced back to a server over the network even if local.  We should be using "%APPDATA%\Local" to ensure it's local and stays local, I think.